cnames_active.js

```diff
@@ -294,6 +294,7 @@ var cnames_active = {
   "awoo": "awoojs.github.io/website",
   "aws-auth": "iamarkadyt.github.io/aws-auth",
   "axii": "ariesate.github.io/axii-site",
+  "axios-c": "Nom-nom-hub.github.io/axios-cache-lite",
```

Comment:
```suggestion
  "axios-cache-lite": "nom-nom-hub.github.io/axios-cache-lite",
```
Could I suggest `axios-cache-lite.js.org` as the subdomain to request, for clarity as it relates to the project itself?

(As the CI error notes, while the path is case-sensitive, the github.io subdomain is not and show be lowercase)